src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/Internal/TopicMessages.cs

```diff
@@ -316,6 +317,25 @@ internal sealed class GossipTick
         private GossipTick() { }
     }
 
+    /// <summary>
+    /// Internal event signalling that a new subscriber has been added to the registry
+    /// either locally using <see cref="Put"/>, <see cref="Subscribe"/>, or from a <see cref="Delta"/>.
+    /// </summary>
+    internal sealed record NewBucketKeysAdded(IEnumerable<string> Topics);
+    
+    /// <summary>
+    /// Container for buffered <see cref="Publish"/> or <see cref="Send"/> messages
+    /// </summary>
+    /// <param name="Message">The original message being buffered</param>
+    /// <param name="Deadline">The deadline where this buffered message should be timed out</param>
+    internal sealed record BufferedMessage(IWrappedMessage Message, Deadline Deadline, IActorRef Sender);
+
+    internal sealed class PruneBufferTick
```

Comment:
New timer message for pruning timed out messages

Reply:
Where does this get handled? Tried looking for it

Reply:
Done